src/controllers/userController.js

```diff
@@ -8,7 +8,7 @@ const register = async (req, res) => {
         
```

Comment:
I think i missed it before but lets make sure they send an email and password, or whatever is required before we even make api calls to the DB.  you can fix this later.